src/vcpkg/install.cpp

```diff
@@ -216,8 +216,8 @@ namespace vcpkg
             bool operator()(const std::string& lhs, const std::string& rhs) { return lhs < rhs; }
             bool operator()(const file_pack& lhs, const file_pack& rhs) { return lhs.first < rhs.first; }
 #endif
-            bool operator()(const std::string& lhs, const file_pack& rhs) { return lhs < rhs.first; }
-            bool operator()(const file_pack& lhs, const std::string& rhs) { return lhs.first < rhs; }
+            bool operator()(const std::string& lhs, const file_pack& rhs) const { return lhs < rhs.first; }
```

Comment:
The other ones should be as well.

Reply:
I don't think that compilers but I will try now